yt_dlp/extractor/twitch.py

```diff
@@ -306,6 +348,26 @@ def _extract_info(info):
 
     @staticmethod
     def _extract_info_gql(info, item_id):
+        def moment_to_chapter(moment):
+            moment = moment.get('node')
+            if moment.get('type') != 'GAME_CHANGE':
+                # TODO: warning or debug log if found unknown moment type
+                return None  # TODO: or empty dict?
```

Comment:
> moments seem to be a multipurpose feature (but for now this might be the only type that exists), and I have a feeling that when Twitch modifies this feature, chapter extraction might silently break and I'll never notice.
probably this is silly, but I see it the way as there is a warning too when a description was not found: it hints that something might be off there.

even if they add/change a new moment type, as long as the momentPosition and duration exists, the chapter will be well-formed and it doesnt matter